src/js/index.js

```diff
@@ -63,7 +64,9 @@ RequestUtil.json = function(options = {}) {
 (function() {
   function renderApplication() {
     function renderAppToDOM(content) {
-      ReactDOM.render(content, domElement, function() {
+      console.log(content);
+      const hotContent = hot(module)(() => content);
```

Comment:
Does this work? I tried something similar while approaching this problem in the first place, but it didn't work for me